docs/templates/singulink/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Singulink
```

Comment:
Is this within our right to update?

Reply:
oh. i honestly don't know. :dumb:
they haven't updated their year in copyright *yet*